src/BenchmarkDotNet/Configs/WakeLockType.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace BenchmarkDotNet.Configs
+{
+    public enum WakeLockType
+    {
+        /// <summary>
+        /// Forces the system to be in the working state while benchmarks are running.
+        /// </summary>
+        RequireSystem,
```

Comment:
Can we drop the "Require"? It reads weird to me. It's already implied by the enum name.
```suggestion
        System,
```